From b7b85808414feeec8f9996cada6e12a8af0e049c Mon Sep 17 00:00:00 2001
From: Shuzhen Wang <shuzhenwang@google.com>
Date: Fri, 18 Oct 2024 17:20:06 +0000
Subject: [PATCH 110/436] Revert "Enable clang-format for camera framework
 native code"

This reverts commit 00526b660e152947ae167ec077a5c68f6b72ce1b.

Reason for revert: Reformatting cameraservice code make git blame hard to use.

Change-Id: Id076a1d3353869ac569799d6c133014ac9fcebdf
---
 PREUPLOAD.cfg | 2 --
 1 file changed, 2 deletions(-)

diff --git a/PREUPLOAD.cfg b/PREUPLOAD.cfg
index 34ff367d6b..e9b757bb4b 100644
--- a/PREUPLOAD.cfg
+++ b/PREUPLOAD.cfg
@@ -30,10 +30,8 @@ bpfmt = -s
 
 # Only turn on clang-format check for the following subfolders.
 clang_format = --commit ${PREUPLOAD_COMMIT} --style file --extensions c,h,cc,cpp
-               camera/
                media/libaudioclient/tests/
                media/libaudiohal/tests/
                media/libmediatranscoding/
                services/camera/virtualcamera/
                services/mediatranscoding/
-               services/camera/libcameraservice/
-- 
2.43.0

